File: TaskManager/TaskManager/TaskManager/Models/Issue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using global::TaskManager.Enums;
using System;
using System.Collections.Generic;
using TaskManager.Models;

namespace TaskManager.Models
{
    public class Issue
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public Priority Priority { get; set; }
        public Status Status { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime LastModifiedDate { get; set; }
        public string AssignedTo { get; set; }
        public List<string> Tags { get; set; }
        public List<Comment> Comments { get; set; }

        public Issue()
        {
            Id = Guid.NewGuid();
            CreationDate = DateTime.UtcNow;
            LastModifiedDate = DateTime.UtcNow;
            Status = Status.Open;
            Tags = new List<string>();
            Comments = new List<Comment>();
        }
    }
}

File: IssueTracker-Final/src/Issue.java
import java.time.LocalDateTime;
import java.util.ArrayList;
import java.util.List;

public class Issue {

    public int Id;

    public String Title;

    public String Description;

    public Priority Priority;

    public  Status status;

    public LocalDateTime Creationdate;

    public LocalDateTime LastModifiedDate;

    public String AssignedTo;

    public List<String> Tags=new ArrayList<>();

    public List<Comment> comments=new ArrayList<>();


    public Issue (int ID,String Title,String Description,Priority Priority, Status Status,LocalDateTime Creationdate,LocalDateTime LastModifiedDate, String assignedTo ,List<Comment> comments,List<String>tags ){

        this.Id=ID;
        this.Title=Title;

        this.Description=Description;

        this.Priority=Priority;

        this.status=Status;

        this.Creationdate=Creationdate;
        this.LastModifiedDate=LastModifiedDate;
        this.AssignedTo=assignedTo;
        this.comments=comments;

        this.Tags=tags;







    }


    public int getId() {
        return Id;
    }






























}
